src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -799,7 +799,11 @@ public void onBuildingMove(final IBuilding oldBuilding)
     @Override
     public boolean canEat(final ItemStack stack)
     {
-        return true;
+        if (stack.getItem().getFood().getHealing() >= getBuildingLevel())
```

Comment:
Is there food that heals 5?

Reply:
4/5 especially

Reply:
I tested with a lvl 5 builder and she ate steak. 1 healing is half a haunch, I think.

Reply:
okay that's fine then.